search_service/elastic/query_strings.py

```diff
@@ -104,6 +104,8 @@ def search_query(text, uid, start_point, synonym_analyzer=FILTER.get("en")):
         },
         '_source': ['textversions.content', 'textversions.statementUid'],
         'highlight': {
+            "pre_tags": ["<strong>"],
```

Comment:
Using `<em>` is the more idiomatic way to achieve the results. With `<em>` you are more flexible in how you highlight the results. With `<strong>` you are hardcoding it to a bold font. I'd suggest to use `<em>` as offered from elasticsearch and then highlight as you want it in the CSS file.

CC @tkrauthoff